app/upgrades.go

```diff
@@ -78,3 +81,23 @@ func v010204UpgradeHandler(app *Quicksilver) upgradetypes.UpgradeHandler {
 		return app.mm.RunMigrations(ctx, app.configurator, fromVM)
 	}
 }
+
+func v010205UpgradeHandler(app *Quicksilver) upgradetypes.UpgradeHandler {
+	return func(ctx sdk.Context, _ upgradetypes.Plan, fromVM module.VersionMap) (module.VersionMap, error) {
+		//update minter epoch-provisions
+		minter := app.MintKeeper.GetMinter(ctx)
+		minter.EpochProvisions = sdk.NewDec(50000000).Quo(sdk.NewDec(365))
+		app.MintKeeper.SetMinter(ctx, minter)
+
+		//update params
+		params := app.MintKeeper.GetParams(ctx)
+		params.DistributionProportions = minttypes.DistributionProportions{
+			Staking:              sdk.NewDec(0.8),
```

Comment:
This is invalid for NewDec; these values will all be implicitly truncated and set to 0, I believe?

You will need `NewDecWithPrec(8, 1)`, etc.